js/index.js

```diff
@@ -0,0 +1,32 @@
+$(document).ready(function() {
+
+$('#submit-btn').click(submitOnClick);
+//  We are now in the button.submit `click` event listener
+  function submitOnClick(event) {
+    event.preventDefault();
+
+    var cityName = $('#city-type').val().trim();
+    cityName = cityName.toLowerCase();
+
+    if (cityName === 'new york city' || cityName === 'new york' || cityName ===  'nyc') {
+      $('body').addClass('nyc').removeClass('sf','la','austin','sydney');
+      $('#city-type').val('');
+    }
+    else if (cityName === 'san francisco' || cityName === 'sf' || cityName ===  'bay area') {
+      $('body').addClass('sf').removeClass('nyc','la','austin','sydney');
```

Comment:
Your syntax for `.removeClass();` should not include those commas or extra quotes. It should look like this: `.removeClass('nyc la austin sydney');`